lsst-epo/investigations-client
# [C] source selector fixes

## What this change does ##

Fixes for Source Selector

- Use preview URL from Canto to get consistent 1000px image
- Display controls always (no more difference in fullscreen and on-page display)
- Unwrap and re-map legacy format data
- Create error boundary around widgets to prevent full page crashes

## Notes for reviewers ##

Client only

## Testing ##

- Inspect in console to see that images in the source selector are fixed at 1000px
- Verify that controls are visible and work
- Upload a legacy format light curve file to a dataset, attach to a widget, and verify the alerts show.

[ZTF19aavjzrz_r_lightcurve.json](https://github.com/lsst-epo/investigations-client/files/15003462/ZTF19aavjzrz_r_lightcurve.json)
